torch/_dynamo/convert_frame.py

```diff
@@ -135,7 +134,6 @@
 except ModuleNotFoundError:
     np = None
 
-
```

Comment:
Looks like the changes to this fille can just be reverted

Reply:
updated